tracktolib/api.py

```diff
@@ -214,3 +221,21 @@ def __init__(self):
 
 def check_status(resp, status: int = starlette.status.HTTP_200_OK):
     assert resp.status_code == status, json.dumps(resp.json(), indent=4)
+
+
+def generate_list_name_model(class_name: str):
+    name = f"List{class_name}"
+    return {
+        "responses": {
+            "200": {
+                "content": {
+                    "application/json": {
+                        "schema": {
+                            "title": name,
```

Comment:
is it possible to override only the title, without overriding the rest ?

Reply:
yes it detect automatically  Array. I remove type declaration

Reply:
What I mean is that you are overriding the full "responses". Let's say I have something defined at "201" or "401", what will happen?